lib/code_corps/map_utils.ex

```diff
@@ -8,18 +8,18 @@ defmodule CodeCorps.MapUtils do
   def keys_to_string(map), do: stringify_keys(map)
 
   # Goes through a list and stringifies keys of any map member
-  def stringify_keys(nil), do: nil
-  def stringify_keys(%DateTime{} = val), do: val
-  def stringify_keys(map = %{}) do
+  defp stringify_keys(nil), do: nil
+  defp stringify_keys(%DateTime{} = val), do: val
```

Comment:
It's not. A module struct matches a map, so without this line here, line 13 would match a `%DateTime{}` and try to recursively stringify those keys as well.